src/screens/conclusion/index.js

```diff
@@ -1,73 +1,85 @@
-import "./conclusion.css";
+import { PatreonIcon } from "../../components/patreon-icon";
 
 export function Conclusion() {
   return (
-    <section id="conclusion" className="together-we-can screen-content">
-      <div className="full-screen bears h-full w-full flex-col justify-between">
-        <div className="h-screen-50vh flex w-[100%] flex-col self-start py-16 pr-10 lg:w-[75%] 2xl:w-[50%]">
-          <h2>Together, we can make life better for wild animals.</h2>
+    <section
+      id="conclusion"
+      className="relative min-h-screen bg-woods bg-cover md:bg-woods-md md:bg-center"
+    >
+      <img
+        className="absolute right-20 top-52 w-96 lg:w-[500px]"
+        src="/images/conclusion/momma.png"
+        alt="Mother bear"
+      />
+      <img
+        className="absolute bottom-72 right-96 w-[500px]"
+        src="/images/conclusion/pups.png"
+        alt="Bear pups"
```

Comment:
Here is a demo of the images for the background woods and the two bear images. I'm not sure if the bear placements going behind the text are perfect yet. For mobile I will change their position later. 
https://user-images.githubusercontent.com/39889198/201736521-0bb21841-73f6-46be-b463-f3b9e7b64f6d.mov